app/matchers/hyku_addons/csv_matcher.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+module HykuAddons
+  class CsvMatcher < Bulkrax::CsvMatcher
+    def parse_date_published(src)
+      date_parts = src.split('-')
```

Comment:
Might it be better to use `date = Date.edtf(str)` and then `date.year` etc?

Reply:
Possibly.  The dates coming are in a specific format instead of the full possibilities of EDTF dates so I _think_  this is safe for now.